src/implementor/models.py

```diff
@@ -0,0 +1,16 @@
+import marshmallow as marsh
+from peewee import CharField
+
+from src.base_model import BaseModel
+
+
+class SimulationConfiguration(BaseModel):
+    """Represents a single simulation configuration."""
+
+    class Schema(BaseModel.Schema):
+        description = marsh.fields.String()
+
+        def _make(self, data: dict) -> "SimulationConfiguration":
```

Comment:
Why the Type Hint as a string and not a class?

Reply:
![Screenshot 2023-04-03 at 09 53 31](https://user-images.githubusercontent.com/45163503/229446243-2437134b-0671-4632-acae-c8ec5bd30f11.png)

Good question 👍 Because I got this error message :/ I think the class is not completely defined at this line of code, but I'm not sure. 😇

Reply:
I see. If I recall correctly, Christian mentioned something like that.

Reply:
Perfect 😇 Thanks for the question!